yarn.lock

```diff
@@ -10654,6 +10691,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"redux@npm:^5.0.1":
```

Comment:
This adds redux 5 without also removing redux 4. Is that the expected behavior for this first stage, or should we expect to only use a single version of redux.

Reply:
> This adds redux 5 without also removing redux 4. Is that the expected behavior for this first stage, or should we expect to only use a single version of redux.

We need to keep both versions for now.
The expected behavior when adding Redux Toolkit is indeed to have Redux 5 added without immediately removing Redux 4. However, the next step in the migration process will be to remove the direct dependency on Redux 4 (and other packages like redux-thunk and reselect) from the package.json, as they're now provided by Redux Toolkit. This should be done only after we have `fully migrated to Modern Redux`.

Source of Information: https://redux.js.org/tutorials/fundamentals/part-8-modern-redux#package-cleanup